review_checklist.md

```diff
@@ -0,0 +1,35 @@
+# Review Checklist
+
+When reviewing, keep in mind that we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our contributions and interactions within this repository.
+
+**General tips**
+- Try create a culture of collaboration, when reviewing
+- Welcome the author and thank them for the pull request
+- When suggesting changes, try to use questions, not statements
+- When suggesting mandatory changes, do it as politely as possible providing documentation references
+- If your suggestion is optional or a matter of personal preferences, please say it explicitly
+- When asking for adding tests or for complex code refactoring, say that the author is welcome to ask for clarifications and help if they need
+- If somebody suggests a good idea, say about it or put a thumbs up
+- After merging, thank the author and reviewers for their time and effort
+
+**Standards and documentation**
+- [ ] if the pull request is not a documentation fix, it must include a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#creating-a-changelog-fragment). Please check the format carefully
```

Comment:
```suggestion
- [ ] if the pull request is not a documentation fix, it must include a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#creating-a-changelog-fragment). Please check the format carefully.
```

Reply:
yes, i know about it. If I add the dot, not sure if the next sentence should start with a capital letter and so on.. let's try another approach

Reply:
PTAL

Reply:
Thanks for the change, it's more convenient now imho.

Reply:
@aminvakil thanks for spotting this and giving the feedback